cluster/manifests/flannel/daemonset.yaml

```diff
@@ -35,7 +35,7 @@ spec:
           - name: CNI_CONFIG_SOURCE
             value: /etc/kube-flannel/cni-conf.json
           - name: CNI_CONFIG_TARGET
-            value: /etc/cni/net.d/10-flannel.conf
+            value: /etc/cni/net.d/10-flannel.conflist
```

Comment:
Does it do the filename change by itself? Because I don't see it anywhere else in this PR.

Reply:
It's just the name of the target file, Our init container copies from `CNI_CONFIG_SOURCE` to `CNI_CONFIG_TARGET`